src/components/MessagesList/MessagesGroup/Message/MessagePart/AudioPlayer.vue

```diff
@@ -72,6 +80,37 @@ export default {
 		},
 	},
 
+	mounted() {
+		document.addEventListener('audioPlayerEnded', (e) => {
```

Comment:
We have EventBus used in the code, which I'd used instead of custom events:

- audio component stops playing and emits event with message id
- all other audio components listen to this event.
- one which message is next from stopped one (could be a computed prop based on messagesStore) stars to play
- repeat from the start